harvard-lts/hdc3a-mock-services
# Hdc114 - Mock Starfish and DRS Ingest Services

**Mocks Starfish and DRS for end-to-end testing of DIMS and DTS**
* * *

**GitHub Issue**: https://github.com/harvard-lts/HDC/issues/114

# What does this Pull Request do?
Allows DIMS and DTS to run without working Starfish and DRS Ingest connections.

# How should this be tested?
Make sure to use the DTS branch: https://github.com/harvard-lts/drs-translation-service/tree/HDC114 to spin up a local instance of DTS.
Follow the README for this branch under the Testing header: https://github.com/harvard-lts/hdc3a-mock-services/tree/HDC114#readme

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? no
- integration tests? no

# Interested parties
@jessjass @awoods @GPortas @qqmyers
